src/test/java/com/future/function/qa/steps/Scoring/Comparison/ComparisonSteps.java

```diff
@@ -17,7 +17,7 @@
   @When("^user click the last final judging data$")
   public void userClickTheLastFinalJudgingData() {
     int lastIndex = judgingListPage.countJudgingDataList();
-    judgingListPage.findJudgingData(lastIndex).click();
+    judgingListPage.findJudgingData(lastIndex-1).click();
```

Comment:
Maybe just pass `lastIndex` and do not add `row` variable in `judgingListPage.findJudgingData` as the code in that function adds the `row` variable with 1 again. See below:

```java
  return find(By.xpath(String.format("//body/div[@class='app']/div[@class='main-container is-fullhd']" +
        "/div[@class='main-content']" +
        "/div[@class='auto-overflow-container']/div[@class='judging-container']/div[%s]", row + 1)));
        "/div[@class='main-content']/div[@class='auto-overflow-container']" +
        "/div[@class='judging-container']/div/div[%s]/div[1]", row + 1)));
  }
```

Reply:
Possible solution is to accept and pass `row` variable as is, without any other tampering with its value